books/free-programming-books-subjects.md

```diff
@@ -238,6 +238,8 @@ Books that cover a specific programming language can be found in the  [BY PROGRA
 * [Readings in Database Systems, 5th Ed.](http://www.redbook.io)
 * [Temporal Database Management](http://people.cs.aau.dk/~csj/Thesis/) - Christian S. Jensen
 * [The Theory of Relational Databases](http://web.cecs.pdx.edu/~maier/TheoryBook/TRD.html)
+* [DATABASE SYSTEMS The Complete Book Second Edition](https://people.inf.elte.hu/miiqaai/elektroModulatorDva.pdf)
```

Comment:
Please, read page 4-7.

> NOTICE
> This work is protected by U.S. copyright laws and is provided solely for the use of college instructors in reviewing course materials for classroom use. Dissemination or sale of this work, or any part (including on the World Wide Web), is not permitted.

> http://infolab.stanford.edu/~ullman/dscb.html

@eshellman, Should we accept this kind of works, which are only for instructors (not sure if partial free, I think, reading official website)? Discarded in the past: #6410 #6349

Reply:
1. This is clearly not an authorized distribution.
2. ALL CAPS is not appropriate for catalog entries except for acronyms and sometimes proper names.